dags/mad_server.py

```diff
@@ -62,18 +42,83 @@
         gke_project_id="moz-fx-data-airflow-gke-prod",
         gke_cluster_name="workloads-prod-v1",
         gke_location="us-west1",
-        aws_conn_id=aws_conn_id,
         env_vars=dict(
-            S3_BUCKET="mad-resources-training",
-            S3_ROOT_TRAINING="datasets",
+            GCS_BUCKET="mad-resources-training",
+            GCS_ROOT_TRAINING="datasets",
+            CLOUD_SERVICE="GCS",
             CUSTOMS_TRAINING_ALLOW_OVERWRITE="True",
             AMO_CRED_ISSUER="{{ var.value.AMO_CRED_ISSUER }}",
             AMO_CRED_SECRET="{{ var.value.AMO_CRED_SECRET }}",
-            **s3_env_vars
         ),
         email=[
             "jklukas@mozilla.com",
             "dzeber@mozilla.com",
-            "alissy@mozilla.com",
+            "gleonard@mozilla.com",
+        ],
+    )
+    mad_train_model = gke_command(
+        task_id="train_model",
+        cmds=[
+            "/bin/bash",
+        ],
+        image_pull_policy='Always',
+        command=[
+            "bin/train_model",
+            "--publish",
+            "--publish-as-latest",
+            "./working",
+        ],
+        docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/mad-server:latest",
+        startup_timeout_seconds=500,
+        gcp_conn_id="google_cloud_airflow_gke",
+        gke_project_id="moz-fx-data-airflow-gke-prod",
+        gke_cluster_name="workloads-prod-v1",
+        gke_location='us-west1',
+        env_vars=dict(
+            GCS_BUCKET="mad-resources-training",
+            GCS_ROOT_TRAINING="datasets",
+            CUSTOMS_TRAINING_ALLOW_OVERWRITE="True",
+            CLOUD_SERVICE="GCS",
+            GCLOUD_PROJECT='mad-model-training',
+            GCS_REPORT_BUCKET='mad-reports',
+        ),
+        email=[
+            "jklukas@mozilla.com",
+            "dzeber@mozilla.com",
+            "gleonard@mozilla.com",
         ],
     )
+    new_data_eval = gke_command(
+        task_id="evaluate_new_data",
+        cmds=[
+            "/bin/bash",
+        ],
+        image_pull_policy='Always',
+        command=[
+            "bin/evaluate_new_data",
+            "--publish",
+            "--publish-as-latest",
+            "./working",
+        ],
+        docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/mad-server:latest",
+        startup_timeout_seconds=500,
+        gcp_conn_id="google_cloud_airflow_gke",
+        gke_project_id="moz-fx-data-airflow-gke-prod",
+        gke_cluster_name="workloads-prod-v1",
+        gke_location='us-west1',
+        env_vars=dict(
+            GCS_BUCKET="mad-resources-training",
+            GCS_ROOT_TRAINING="datasets",
+            CUSTOMS_TRAINING_ALLOW_OVERWRITE="True",
+            CLOUD_SERVICE="GCS",
+            GCLOUD_PROJECT='mad-model-training',
+            GCS_REPORT_BUCKET='mad-reports',
```

Comment:
There's a good bit of duplication of configuration between these 3 tasks. It may be worth factoring out some of this to reusable constants.

Reply:
Update and re-rested